app/controllers/api/v2/hosts_controller.rb

```diff
@@ -197,6 +197,34 @@ def get_status
         end
       end
 
+      api :DELETE, "/hosts/:id/status/:type", N_("Clear sub-status of host")
+      param :id, :identifier_dottable, :required => true
+      param :type, [ HostStatus::Global ] + HostStatus.status_registry.to_a.map { |s| s.humanized_name }, :required => true, :desc => N_(
```

Comment:
It actually shouldn't contain global right? It generates following in apipie doc

```
Must be one of: HostStatus::Global, configuration, build.
```

Reply:
Fixed!